[JA] translate nft concept

tequdev opened the pull request:
- nftoken-batch-minting
- nftoken-auctions
- nftoken-authorized-minting

Commit:
[JA] new translate nftoken-batch-minting.ja

Commit:
[JA] new translate nftoken-auctions

Commit:
[JA] new translate nftoken-authorized-minting

Commit:
fix some translation

Commit:
Merge commit '1740f3e72bbd85c2167405ca97d5e5cf3a75b2b7' into japanese-nft-concept

mDuo13 commented on content/concepts/tokens/nftoken-auctions.ja.md:
```diff
@@ -0,0 +1,64 @@
+parent: non-fungible-tokens.html
+blurb: NFTのMintを他のアカウントに代行してもらうことができます。
+labels:
+ - 非代替性トークン, NFT
```
This is OK, but it'll show up as gray and not link to a landing page like the English tag unless we do some further edits. We need to add a landing page entry for it in `dactyl-config.yml` and modify the SCSS/CSS to pick a color for it as well. I can do that separately after merging this PR, though.

mDuo13 commented on content/concepts/tokens/nftoken-auctions.ja.md:
```diff
@@ -0,0 +1,64 @@
+labels:
+ - 非代替性トークン, NFT
+---
+
```
Leaving an empty line between the frontmatter and the page title _might_ result in some issues with autodetecting the title. I suggest deleting the empty line just to be safe. (Though, it's probably safe since the English version does this too.)

mDuo13 commented on dactyl-config.yml:
```diff
@@ -812,18 +812,27 @@ pages:
+        targets:
             - ja
 
     # TODO: translate
```
Don't forget to remove the TODOs since you've done it (on line 823 also)

mDuo13 commented on content/concepts/tokens/nftoken-auctions.ja.md:
```diff
@@ -0,0 +1,64 @@
+
+最低落札価格ありのオークションとして、ブローカー方式で運営する。
+
+![Auction in Brokered Mode with a Reserve](img/nft-auction1.png "Auction in Brokered Mode with a Reserve")
```
You can safely translate the alt text and title text for images. For example:

```suggestion
![ブローカー方式で最低落札価格ありのオークション](img/nft-auction1.png "ブローカー方式で最低落札価格ありのオークション")
```

mDuo13 reviewed (approved):
Largely looks good although I left a couple minor notes

tequdev committed:
Update content/concepts/tokens/nftoken-auctions.ja.md

Co-authored-by: Rome Reginelli <mduo13@gmail.com>

Commit:
fix some points

- empty line between the frontmatter and the page title
- Remove TODOs for translation
- alt text and title text for images

Commit:
Merge commit '09512ed47ce5e331dd6a9f08c2f706620fa19313' into japanese-nft-concept

tequdev commented:
The points raised have been corrected, but I will also translate the following pages, which are newly added as nft concepts.

- concepts/tokens/nft-reserve-requirements.md
- concepts/tokens/nft-collections.md
- concepts/tokens/nft-fixed-supply.md

Commit:
fix empty line between the frontmatter and the page title

Commit:
[JA] nft-reserve-requirements

Commit:
[JA] nft-collections

Commit:
[JA] nft-fixed-supply

- add nft-fixed-supply to dactyl-config.yml

Commit:
fix

tequdev commented:
Added the nft-fixed-supply page to dactyl_config.yml as it had not been added.
